lib/src/main/java/com/adyen/nexoapp/lib/model/api/body/payment/PaymentData.kt

```diff
@@ -0,0 +1,7 @@
+package com.adyen.nexoapp.lib.model.api.body.payment
+
+import com.squareup.moshi.Json
+
+data class PaymentData(
+    @Json(name = "PaymentType")    val timeStamp: String
```

Comment:
Incorrect name.

Reply:
What do you mean? This object and element is needed for the refund by card.